signadot/operator/values.yaml

```diff
@@ -103,25 +103,53 @@ ioContextServer:
   # imagePullPolicy: Always
 
 # routeInit:
+#   resources:
+#     limits:
+#       cpu: 100m
+#       memory: 128Mi
+#     requests:
+#       cpu: 60m
+#       memory: 64Mi
 #   image: my-private-registry/signadot/route-sidecar-init:latest
 #   legacy:
 #     image: my-private-registry/signadot/sd-init-networking:latest
 #   imagePullPolicy: Always
 #   imagePullSecret: secret-routeInit
 
 # routeSidecar:
+#   resources:
+#     limits:
+#       cpu: 100m
+#       memory: 128Mi
+#     requests:
```

Comment:
What's specified here doesn't seem like what gets applied by default in the installation. I see:

```
   resources:
      limits:
        cpu: "2"
        memory: 1Gi
      requests:
        cpu: 100m
        memory: 128Mi
```

Is this intended? I think it would make more sense to comment out the default values here like we do elsewhere right? @daniel-de-vera

Reply:
Injected containers have a default value in code (defined by controller manager), that's why it's not added here. But if you want, sure we can update and comment out those values here.

Reply:
Yeah, I think that would make sense if the commented values are the default, it gives a clear reference point when people want to change it.

Reply:
I can take care of this update by copying it from the readme if the readme is already correct @daniel-de-vera

Reply:
Yes, the readme values should be correct.